### ISSUE TITLE
How to expose only a few fields from model to api, if I leverage t.crud

### ISSUE THREAD

**Issue by psypath** [graphql-nexus/nexus-plugin-prisma#348]
What should I do if I want to expose only a few fields from model to api? Example:

In prisma I have:
```
model Email {
  id                            ID
  value                         String      @unique
  expirationMoment  DateTime
}
```
`expirationMoment` should be set to `now + 1 day` (in code) and should not be exposed by graphql api neither by output, nor by input objects.

I can hide `expirationMoment` from being a part of `Email` output type using this:

```
export const Email = objectType({
    name: "Email",
    definition(t) {
        t.model.id()
        t.model.value()
    },
})
```

but if I expose CRUD operations using this:

```
export const EmailMutation = extendType({
    type: "Mutation",
    definition(t) {
        t.crud.createOneEmail({alias: "createEmail"})
    },
})
```

then EmailCreateInput type will contain a required `expirationMoment` anyway.

How to avoid this?

**Comment by jasonkuhrt** [comment]
I think it could be intuitive if the way which model is used affects the related crud methods.

**Comment by Weakky** [comment]
I don't think models and inputs should be coupled, it entirely depends on what you're trying to achieve. eg:

One could be willing to expose `expirationMoment` on `Email`, but not on the input type because you want it to always be programmatically defined.

I added that issue to #477, as we need to offer a way to customize input types (namely `EmailCreateInput` here)

**Comment by jasonkuhrt** [comment]
@Weakky agree, it is more nuanced than I make it seem. We might want the concept of strategies/polices.
